### PR TITLE
SAK-29362 - Forums: Display availability dates

### PR DESCRIPTION
**Pull request by mcarropellicer**
SAK-29362 - Forums: Display availability dates

### CODE REVIEW

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -3372,6 +3372,10 @@
 # DEFAULT: true
 # msgcntr.forums.exclude.deleted.onlywithoutdescendant=false 
 
+# SAK-29362 Display the availability dates in forums and topics.
+# DEFAULT: false
+# msgcntr.display.availability.dates = false
```
Comment (juanjmerono) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: This should be "true" the opposite to the default value.

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java [hunk 2]
```diff
@@ -38,6 +38,9 @@
 import org.sakaiproject.api.app.messageforums.ui.DiscussionForumManager;
 import org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager;
 
+import org.sakaiproject.time.cover.TimeService;
```
Comment (juanjmerono) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java]: Could you avoid the use of cover service here? Maybe loading through the ComponentManager service.

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties (hunk 1 above)
Comment (mcarropellicer) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Done, thanks Juanjo.

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java (hunk 2 above)
Comment (mcarropellicer) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java]: I agree with you, we should avoid the use of covers, but to be honest MSGCNTR uses the covers widely in several classes, i think this concern is outside the scope of this patch.

**Comment by juanjmerono** [comment]
Please squash commits into a single one following this guide https://confluence.sakaiproject.org/display/SAKDEV/Git+Setup

Thanks Miguel.

**Comment by mcarropellicer** [comment]
Squash done.

**Comment by ottenhoff** [comment]
Should the default be true instead?  This seems like good functionality that most people want.

**Comment by mcarropellicer** [comment]
Agree, it's a new functionality and i don't know if all institutions agree, so the default value was set to false to respect the current OOTB behaviour.

Miguel

**Comment by juanjmerono** [comment]
+1, I can merge this PR and open a new Jira to change the default value to true. Sounds good?

**Comment by ottenhoff** [comment]
I would prefer we just change the default now, squash the commit, and git push -f

**Comment by mcarropellicer** [comment]
The default value is true, also changed the value in the default properties.

Miguel
